packages/stryker-elements/src/lib/atoms/loader.ts

```diff
@@ -1,22 +1,67 @@
 import { html } from 'lit';
-import { classMap } from 'lit/directives/class-map.js';
 import { customElement, property } from 'lit/decorators.js';
-
+import './spinner';
 import { BaseElement } from '../base-element';
+import { classMap } from 'lit/directives/class-map.js';
 
 @customElement('sme-loader')
 export class Loader extends BaseElement {
+  private _doneWithLoading = false;
```

Comment:
```suggestion
  #doneWithLoading = false;
```

Reply:
Why can't we use the doneWithLoading property instead of a private backing field?